### DISCUSSION TITLE
Sudden high traffic with the Hobby plan, will it be okay?

### DISCUSSION THREAD

**Discussion by stackoverprof** [vercel/vercel#5707] [Help] [answered]
I have no team and with the hobby plan, will there be any problem if a high traffic jump happened, like if I build something (with nextjs and hosted in vercel) that will hit a huge number of participant suddenly

**Comment by styfle** [comment, marked as answer]
Hi @stackoverprof 

Take a look at the [Fair Use Policy](https://vercel.com/docs/platform/fair-use-policy) find more details about what is or is not allowed.

If you aren't violating this policy or exceeding any of the limits, then it shouldn't be a problem.

If you still have questions about your specific use case, its best to [Contact Support](https://vercel.com/support/request).
